include/wx/uilocale.h

```diff
@@ -56,20 +57,28 @@ class WXDLLIMPEXP_BASE wxLocaleIdent
     // Set modifier (only supported under Unix)
     wxLocaleIdent& Modifier(const wxString& modifier);
 
+    // Set sortorder (only supported under Windows)
+    wxLocaleIdent& Extension(const wxString& extension);
+
+    // Set sortorder (only supported under Windows)
+    wxLocaleIdent& Sortorder(const wxString& sortorder);
+
     // Accessors for the individual fields.
     const wxString& GetLanguage() const { return m_language; }
     const wxString& GetRegion() const { return m_region; }
     const wxString& GetScript() const { return m_script; }
     const wxString& GetCharset() const { return m_charset; }
     const wxString& GetModifier() const { return m_modifier; }
+    const wxString& GetExtension() const { return m_extension; }
+    const wxString& GetSortorder() const { return m_sortorder; }
 
     // Construct platform dependent name
     wxString GetName() const;
 
     // Get the language tag: for the objects created with FromTag() returns the
     // string passed to it directly, otherwise reconstructs this string from
     // the components.
-    wxString GetTag() const;
+    wxString GetTag(wxLocaleTagType tagType = wxLOCALE_TAGTYPE_DEFAULT) const;
```

Comment:
I think it might make sense to rename this to `MakeTag()` or `BuildTag()` as it's not a trivial accessor any longer.

Reply:
> I think it might make sense to rename this to `MakeTag()` or `BuildTag()` as it's not a trivial accessor any longer.

Well, IMHO the method name `GetTag` is just fine. If the `wxUILocaleIdent` was constructed via method `FromTag` the tag is returned _as is_ - that is nothing is explicitly made or built, and if the default or system tag type is chosen, the method simply calls method `GetName` - again nothing is explicitly made or built. The method parameter `tagType` allows the user to influence how the tag is formatted, if the platform dependent tag type is not suitable for the application.

Reply:
OK, let's leave it like this then.